config/config.go

```diff
@@ -196,13 +196,13 @@ type Coordinator struct {
 		// GasPriceIncPerc is the percentage increase of gas price set
 		// in an ethereum transaction from the suggested gas price by
 		// the ethereum node
-		GasPriceIncPerc int64
+		GasPriceIncPerc int64 `validate:"gte=0"`
```

Comment:
if the field is not `required`, the `gte=0` validation does not check, try to set this value to zero and run, the validation goes to pass

Reply:
maybe we can remove it, or use some custom validations

Reply:
Actually, gte is >= 0, not > 0, so zero should work